src/app/(summit-pages)/blocks/agenda.tsx

```diff
@@ -35,56 +35,61 @@ export const Agenda = () => {
         </Content>
       </Wrapper>
       <ul className="col-span-full">
-        {talks?.map((talk) => {
-          const start = new Date(talk.startTime)
-
-          return (
-            <li
-              key={talk._id}
-              className="group cursor-pointer border-b border-solid border-blue-500 px-2 py-8 first:border-t hover:bg-gray-50 dark:border-rose-500 dark:hover:bg-indigo-950"
-            >
-              <Link
-                href={getPermalinkFromTalk(talk)}
-                onClick={getOnClickHandler(talk)}
+        {talks
+          ?.sort((talk1, talk2) => {
+            const startTime1 = new Date(talk1.startTime).getTime()
+            const startTime2 = new Date(talk2.startTime).getTime()
+            return startTime1 - startTime2
+          })
```

Comment:
nice. i think @mszekiel recommended this package once https://github.com/iamkun/dayjs - its just 6kb 👀